students/dreifler/Lesson2/FizzBuzz.py

```diff
@@ -0,0 +1,15 @@
+#FizzBuzz
+
+def fizzBuzz():
+    for a in range (1, 100):
```

Comment:
should be `range(1, 101)` to include 100 number.

also per pep8 style guidelines variable and function names follow snake_case convention -> `def fizz_buzz`